pkg/api/grpc/grpc.go

```diff
@@ -501,6 +501,20 @@ func (a *api) InvokeBinding(ctx context.Context, in *runtimev1pb.InvokeBindingRe
 		req.Metadata[key] = val
 	}
 
+	// this is for the http binding, so dont need grpc-trace-bin
+	if span := diagUtils.SpanFromContext(ctx); span != nil {
+		sc := span.SpanContext()
+		tp := diag.SpanContextToW3CString(sc)
```

Comment:
can you move the `tp` declaration to inside the `if` statement?

Reply:
I think you mean outside?